### ISSUE TITLE
Facepile] Overflow Button Text Color Unconfigurable

### ISSUE THREAD

**Issue by QuSimone** [microsoft/fluentui#21157]
### Environment Information

- **Package version(s)**: "@fluentui/react": "^7.174.0" <- Please ensure a fix is available for Fluent v7, we are not yet on v8.
- **Browser and OS versions**: Observed on all

### Please provide a reproduction of the bug in a codepen: https://codesandbox.io/s/facepile-tooltip-onfocus-forked-rjico?file=/src/App.tsx

Look at the `overflowButtonStyles` object. You can change `backgroundColor` and see the overflow icon change color. However, the color of the text inside the button remains unconfigurable. 

#### Actual behavior:

![image](https://user-images.githubusercontent.com/72839029/148105655-d7fa617d-2745-4ac9-82d4-e78f3e406666.png)
^ The background color of the overflow button is configurable. The color of the text inside this button is not.

![image](https://user-images.githubusercontent.com/72839029/148105714-41207008-df60-4861-805a-44ee36fe46d3.png)
Utilizing FacePile, custom styles are passed to overflowButton via overflowButtonProps.

![image](https://user-images.githubusercontent.com/72839029/148105798-dca9859a-d6af-4b56-a8d0-b805abea0c88.png)
![image](https://user-images.githubusercontent.com/72839029/148105823-0d93a58f-498a-42be-9bd7-22d1a376c2df.png)
Child of overflowButton is not receiving custom styles. Specifically, this refers to the color of the text inside the overflowButton.  

I believe this file is especially relevant to this issue: https://github.com/microsoft/fluentui/blob/3dbb57b1e6940df2d4039f61a35d961bcb8560c3/packages/react/src/components/Facepile/Facepile.styles.ts

Look at the `overflowInitalsIcon` object. It has a hardcoded color value for the HighContrastSelector.

![image](https://user-images.githubusercontent.com/72839029/148106453-2186123e-0f15-495b-8ac5-1f19acff62d5.png)

#### Expected behavior:

The color of the text inside the overflow button should be configurable.

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No): No

Requested priority: (Blocking, High, Normal, Low) 

Products/sites affected: (if applicable)

**Comment by spmonahan** [comment]
As noted in the issue the problem is that we can override styles for the overflow button but not its children. In this case the child of the overflow button is a `PersonaCoin` and the reason the background color is configurable is that the `PersonaCoin`'s background in `transparent`. However, a text color is specified for `PersonaCoin` which has greater specificity than styles set on the overflow button.

In `Facepile` there are two cases where this is an issue: [when rendering an overflow icon and when rendering overflow description text](https://github.com/microsoft/fluentui/blob/7.0/packages/office-ui-fabric-react/src/components/Facepile/Facepile.base.tsx#L226-L268). Ultimately [these two functions render the content that cannot be overridden](https://github.com/microsoft/fluentui/blob/7.0/packages/office-ui-fabric-react/src/components/Facepile/Facepile.base.tsx#L299-L317).

**Comment by spmonahan** [comment]
This issue also exists in v8 `Facepile`.

**Comment by spmonahan** [comment]
After discussing this with my team @khmakoto showed me how styling overrides work in Fluent UI v7 and v8.

[This example Codesandbox shows how to apply custom styles to Facepile](https://codesandbox.io/s/facepile-with-custom-styles-forked-mj41l?file=/src/App.tsx:970-1137).

More details are available in the [Fluent UI Wiki Component Styling page](https://github.com/microsoft/fluentui/wiki/Component-Styling).
